src/Microsoft.ML.StandardLearners/Standard/MultiClass/MultiClassNaiveBayesTrainer.cs

```diff
@@ -43,24 +43,21 @@ public sealed class Arguments : LearnerInputBaseWithLabel
         {
         }
 
-        private MultiClassNaiveBayesPredictor _predictor;
-
         public override PredictionKind PredictionKind => PredictionKind.MultiClassClassification;
 
-        public override bool NeedNormalization => false;
-
-        public override bool NeedCalibration => false;
-
-        public override bool WantCaching => false;
+        public override TrainerInfo Info { get; }
 
         public MultiClassNaiveBayesTrainer(IHostEnvironment env, Arguments args)
             : base(env, LoadName)
         {
+            Host.CheckValue(args, nameof(args));
```

Comment:
Silly question, but why do we have the `args` parameter?  I don't see it being used.  Am I missing something?

Reply:
No, the class itself is empty, since this has no settings, *except* in entry-point land. I *guess* for consistency?